Corona-Warn-App/build.gradle

```diff
@@ -237,6 +237,13 @@ dependencies {
     implementation 'android.arch.lifecycle:extensions:1.1.1'
     implementation 'androidx.annotation:annotation:1.1.0'
 
+    // DAGGER
```

Comment:
Trying to build the `deviceForTesters` variant fails because it includes another file using the risk-level transaction that has not been refactored.